x-pack/plugins/ml/public/application/util/custom_url_utils.test.ts

```diff
@@ -405,6 +410,58 @@ describe('ML - custom URL utils', () => {
       );
     });
 
+    test('return expected url for Security app', () => {
+      const urlConfig = {
+        url_name: 'Hosts Overview by process name',
+        url_value:
```

Comment:
Probably worth adding a test for the slightly more complicated 'Host details by process name' custom URL, which uses `host.name` and `process.name`:

```
      {
        "url_name": "Host Details by process name",
        "url_value": "security/hosts/ml-hosts/$host.name$?_g=()&query=(query:'process.name%20:%20%22$process.name$%22',language:kuery)&timerange=(global:(linkTo:!(timeline),timerange:(from:'$earliest$',kind:absolute,to:'$latest$')),timeline:(linkTo:!(global),timerange:(from:'$earliest$',kind:absolute,to:'$latest$')))"
      },
```

Reply:
added in https://github.com/elastic/kibana/pull/76957/commits/e8b935e9acc3d287a57b27b6512d4754b25a54bf